libs/designsystem/src/lib/components/list/list.component.scss

```diff
@@ -128,6 +129,7 @@ ion-item-divider {
 
   .item-content {
     display: inline-grid;
+    /* TODO: This min-width is shared*/
```

Comment:
It kinda is. But had to give the other button a 92px min width, to get it to behave visually as this one does. Look into that. Is there a way to share the min-width?

Reply:
Just unresolving this to make sure you are aware of this todo in the code. Feel free to resolve again!

Reply:
Lol thank you. Totally forgot to remove it 🤦